XLPandXeLaTeX.ltx

```diff
@@ -46,74 +46,69 @@ producing complex documents in beautiful formats. \TeX
 
-Unlike most editing programs which are based on either the WYSIWYG paradigm or as text editors used to code or produce markdown, XLingPaper (via the XMLmind XML Editor)
-is a structured editor. Rather than making the document look the way
+Unlike most editing programs which are based on either the \WYSIWYG paradigm or as text editors used to code or produce markdown, XLingPaper (via the \XMLmind \XML Editor)
+is a structured editor. Rather than visually structuring the document look the way
 it is to be formatted, the author “marks up” the items in the document
-according to their kind.  One of the many values this gives is that
+according to their styling needs.  One of the many values this gives is that
 there is a “grammar” of what a well-formed linguistic document looks
-like.  This makes it easy to move and shift sections and chapters and
-examples and prevents users from inadvertently creating ill-formed
+like.  This makes moving, replacing, switching, or reordering sections, chapters, and
+examples less error prone becasue it prevents users from inadvertently creating ill-formed
 documents.
 
-This paper is about the \XeLaTeX{}-produced \PDF{} output.  
-
-The official website for XLingPaper is
-\tbsurl{https://software.sil.org/xlingpaper}.  A list of advantages for the
-author, the reader, and the publisher can be found at the “Why
-XLingPaper?”\cite{black:2017} portion of the documentation.
+The following sections of this paper discuss the\TeX{} technologies used. 
 
 \section{XLingPaper and \TeX{}}
 
-We now describe how XLingPaper uses \TeX{} technologies.
+Due to the heavy reliance on Unicode in modern language documentation and linguistic work, XLingPaper specifically uses \XeLaTeX{} and compatible packages to produce \PDF{} outputs.  The following sections provide more detial on the design requirments and packages used.
 
 \subsection{Design desiderata for \TeX{} with XLingPaper}
 
-From the outset, XLingPaper was designed to be free.  The XMLmind XML Editor had a Personal
-Use License that fit the bill for the vast majority of the target
+From the outset, XLingPaper was designed to be free.  The XMLmind XML Editor had a Personal Use License that fit the bill for the vast majority of the target
 audience of XLingPaper.  The few that did not meet the terms of that
 license most likely would be able to afford to purchase (or have their
 organization purchase) a professional version of the XMLmind XML
 Editor.  The actual XLingPaper plugin has always been free.
 
-Prior to 2009 XLingPaper used RenderX{\footnote{\url{http://www.renderx.com}} to produce \PDF{} documents. However, in 2009 plans were made to add \XeLaTeX-based output to XLingPaper because while there was a free version of
+Prior to 2009 XLingPaper used RenderX{\footnote{\url{http://www.renderx.com}} to produce \PDF{} documents. However, in 2009 plans were made to add \XeLaTeX-based output to XLingPaper because, while there was a free version of
```

Comment:
Good catch!

Reply:
This is about where I stopped. I plan on picking it up some more tonight.